lighthouse-core/test/lib/page-functions-test.js

```diff
@@ -177,16 +180,21 @@ describe('Page Functions', () => {
       assert.equal(pageFunctions.getNodeLabel(el), Array(78).fill('a').join('') + '💡…');
     });
 
-    it('Uses tag name for html tags', () => {
+    it('Returns selector as fallback if nodeLabel equals html tag name', () => {
+      // define a global `window` object for the first
+      // few lines in `getNodeDetails`
+      global.window = {};
```

Comment:
this (plus removing it) should be in the beforeAll/afterAll functions)

Reply:
just pushed, i wasn't entirely sure if we should define `window` from the DOM in beforeAll like other global variables, so I will standby and quickly implement your changes if that definition should be changed.

Reply:
added to beforeAll/afterAll, this will definitely help in writing other getNodeDetail() tests as now there is a `window` variable in the global scope, nice!